lib/objects/objectsInMemFileDB.js

```diff
@@ -2569,6 +2574,52 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
             this.writeTimer = null;
         }
     }
+
+    isEquivalentValue(a, b, ignoreField) {
```

Comment:
I would extend this to "ignoreFields" (plural)

Reply:
But then I need to struglle around with an array here ... ad also today it only works with "first level fields" ... else we need to parse strings

Reply:
The array is not that bad, you just need to .filter out the entries from the property names arrays. 

The deep fields thing is more complicated. We already have these methods: https://github.com/ioBroker/ioBroker.js-controller/blob/8c5fbb98ae1523374332798ccca2cfafe28c646b/lib/cli/cliObjects.js#L396
but I think that's not enough for you here. Especially if you need to exclude the special cases (ts, common.enabled, ...)